functions/Remove-DbaCustomError.ps1

```diff
@@ -0,0 +1,104 @@
+
+        Website: https://dbatools.io
+        Copyright: (c) 2021 by dbatools, licensed under MIT
+        License: MIT https://opensource.org/licenses/MIT
+
+    .LINK
+        https://dbatools.io/Remove-DbaCustomError
+
+    .EXAMPLE
+        PS C:\> Remove-DbaCustomError -SqlInstance localhost, serverName2 -MessageID 70001 -Language "French"
+
+        Removes the custom message on the localhost and serverName2 instances with ID 70001 and language French.
+
+    .EXAMPLE
+        PS C:\> Remove-DbaCustomError -SqlInstance localhost, serverName2 -MessageID 70001 -Language "All"
+
+        Removes all custom messages on the localhost and serverName2 instances with ID 70001.
+    #>
+    [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Low')]
+    param (
+        [parameter(Mandatory, ValueFromPipeline)]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [ValidateRange(50001, 2147483647)]
+        [int32]$MessageID,
+        [String]$Language = 'English',
+        [switch]$EnableException
+    )
+
+    process {
+        $removedMessages = @()
+
+        foreach ($instance in $SqlInstance) {
+            try {
+                $server = Connect-SqlInstance -SqlInstance $instance -SqlCredential $SqlCredential -AzureUnsupported
+            } catch {
+                Stop-Function -Message "Error occurred while establishing connection to $instance" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
+            }
+
+            if ($Language -ine "All") {
+                $languageDetails = $server.Query("SELECT TOP 1 name, alias, msglangid FROM sys.syslanguages WHERE name = '$Language' OR alias = '$Language'")
+
+                if ((Test-Bound Language) -and $null -eq $languageDetails) {
+                    Stop-Function -Message "$server does not have the $Language installed" -Target $instance -Continue
+                }
+
+                $languageName = $languageDetails.name
+                $languageAlias = $languageDetails.alias
+                $langId = $languageDetails.msglangid
+            }
+
+            if ($Pscmdlet.ShouldProcess("Removing server message with id $MessageID from $server")) {
+                Write-Message -Level Verbose -Message "Removing server message with id $MessageID and language $Language from $server"
+                try {
+                    # find the message using language or languageID or the 'session language' message if they specified 'all'. SMO will drop all related messages for an ID if the english message is dropped.
+                    $userDefinedMessage = $server.UserDefinedMessages | Where-Object { $_.ID -eq $MessageID -and ($_.Language -in $languageName, $languageAlias -or $_.LanguageID -eq $langId -or ($Language -ieq "All" -and $_.Language -like "*english")) }
+                    $userDefinedMessage.Drop()
+                    $removedMessages += $userDefinedMessage
```

Comment:
This can be removed. We do not generally output data in the remove commands. Just drop and if successful return nothing.

Reply:
ok